news/2019-08-01-cwc-2019-preview-grand-final.md

```diff
@@ -0,0 +1,43 @@
+Five weeks of frantic fruit catching have been and gone, and the field has been whittled down to the final three. From thirty-two teams, only Chile, China and South Korea remain. All have stood on the top step of the podium before, but who will rise to the occasion to claim this year's greatest prize in osu!catch?
+
+![](/wiki/shared/news/banners/CWC_2019_banner.jpg)
+
+A little over a month ago, the curtain was raised on the 2019 edition of the osu!catch World Cup, with over one hundred and eighty competitors showcasing their dashing skills - all in the hopes of becoming the next _osu!catch Champions_. Seventy-six matches, several nail-biting finishes, and countless incredible moments later, and only three teams remain.
+
+If you missed any of the exciting action that led us to this point, VODs from throughout the tournament are available on the [osu!live Twitch channel](https://www.twitch.tv/osulive) for your viewing pleaseure!
+
+<iframe src="https://player.twitch.tv/?autoplay=false&video=v458642937" frameborder="0" allowfullscreen="true" scrolling="no" height="378" width="100%"></iframe><br/>
+
+I highly recommend checking out the Finals Week 1 matches that led us to here if you have the time, but if you prefer the busy person's overview of last weekend's encounters, feel free to read on below!
+
+## Finals Week 1
+
+This year's Finals opened with a re-ignited conflict as storied as osu!catch itself, with China facing South Korea in the Winner's Bracket Final. Both teams are three-time winners of the osu!catch World Cup, and both of their rosters boast a frankly terrifying array of talent. South Korea started strongly by focusing on slower reading challenge picks from the Hidden pool, knowing that China would undoubtedly focus again on faster DoubleTime and HardRock picks, opening a 3-0 lead. China would retaliate with two points of their own, but could never again get the better of the Korean roster, eventually losing 7-2 in a series of close finishes - never once did the final margin of victory exceed 250,000 points, and in two picks it was one tenth of this figure.
+
+Chile then took a surprising victory of last year's runners-up The United States. Despite losing the opening point, fortune favoured the Chilean squad who repeatedly picked Hidden - a traditional strength of the US - and managed to squeak through by the tiniest of margins, winning each of theif first three picks by less than 35,00 points. The Unites States had avoided HardRock and DoubleTime wherever possible in their previous rounds, and without Hidden to fall back on, their confidence seemed to wane. A last throw of the dice on the HardRock pick [Halozy - Deconstruction Star](https://osu.ppy.sh/beatmapsets/319217#fruits/710613) would have seen them snatch a second game by just 5,000 points, if not for a tragic last-second fail from [-Kurisu-](https://osu.ppy.sh/users/500696) that ensured a 7-1 victory and the chance of another podium placing for last year's 3rd place team.
```

Comment:
```suggestion
Chile then took a surprising victory of last year's runners-up The United States. Despite losing the opening point, fortune favoured the Chilean squad who repeatedly picked Hidden - a traditional strength of the US - and managed to squeak through by the tiniest of margins, winning each of theif first three picks by less than 35,000 points. The United States had avoided HardRock and DoubleTime wherever possible in their previous rounds, and without Hidden to fall back on, their confidence seemed to wane. A last throw of the dice on the HardRock pick [Halozy - Deconstruction Star](https://osu.ppy.sh/beatmapsets/319217#fruits/710613) would have seen them snatch a second game by just 5,000 points, if not for a tragic last-second fail from [-Kurisu-](https://osu.ppy.sh/users/500696) that ensured a 7-1 victory and the chance of another podium placing for last year's 3rd place team.
```

Reply:
Yadda yadda, fixed wrong number and a typo here.

Reply:
Fixed things and also changed first "of" to "over"